### DISCUSSION TITLE
Documentation on how to add a host address

### DISCUSSION THREAD

**Discussion by blachawk** [statiqdev/Discussions#26] [Q&A] [answered]
Hello, 
I was able to get an initial install setup of 1.0.0.beta 11,  This was an easy setup process, and can now see how my content from my input folder compiles into my output folder.  very awesome!

When I run the following from my terminal, I can see my project at `http://localhost:5080/`
  > dotnet run -- preview

Is there any way I can also load `http://local.statiq` in my browser, and see my project there too?

* I have the following line item in my Windows Host file (hosts)
   >127.0.0.1 local.statiq # playing with Statiq Web

* I have IIS Express installed

What should be my next steps?

**Comment by daveaglick** [comment, marked as answer]
Great question! The hosts file entry gets you part of the way there by mapping `local.statiq` to your local network. The other part is going to be the port that the Statiq preview server is listening on.

If you leave the port at the default of "5080" then you should be able to hit the site at `http://local.statiq:5080`. Since DNS resolution (the hosts file bit) and binding a TCP listener to a port are separate things, there's unfortunately no way to add the port to the hosts file.

The other option is to change the default port for the Statiq preview server to the default HTTP port 80 so that you don't need to specify one. This gets tricky because there's a chance you have something else bound to port 80 on your system (and only one process can bind to a port at a time). Try `dotnet run -- preview --port 80` and if it doesn't complain you should be good to go to access it at `http://local.statiq` without specifying a port.

Also, for what it's worth, the Statiq preview server is entirly built-in (it's an in-process [Kestrel server](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1#:~:text=Kestrel%20web%20server%20implementation%20in%20ASP.NET%20Core.%20Kestrel,be%20supported%20on%20macOS%20in%20a%20future%20release.)). You don't need to worry about your IIS Express instance unless you want to serve your output folder through IIS.

**Reply by blachawk** [reply]
Awesome - Many thanks!
